Patch to make sure the fontconfig from EasyBuild will be found
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/reportlab-3.6.12.orig/setup.py b/reportlab-3.6.12/setup.py
index 4b380b6..2ccfa93 100644
--- a/reportlab-3.6.12.orig/setup.py
+++ b/reportlab-3.6.12/setup.py
@@ -277,14 +277,11 @@ class inc_lib_dirs:
             # darwin ports installation directories
             aDir(L, "/opt/local/lib")
             aDir(I, "/opt/local/include")
-        aDir(I, "/usr/local/include")
-        aDir(L, "/usr/local/lib")
-        aDir(I, "/usr/include")
-        aDir(L, "/usr/lib")
-        aDir(I, "/usr/include/freetype2")
         if addrSize==64:
-            aDir(L, "/usr/lib/lib64")
-            aDir(L, "/usr/lib/x86_64-linux-gnu")
+            freepath = os.getenv("EBROOTFREETYPE")
+            aDir(L, pjoin(freepath, "lib64"))
+            aDir(I, pjoin(freepath, "include/freetype2"))
+
         else:
             aDir(L, "/usr/lib/lib32")
         prefix = sysconfig.get_config_var("prefix")
